s32/mcux/devices/S32Z270/S32Z270_device.h

```diff
@@ -90,760 +90,752 @@ typedef struct {
 
-#define CAN_CBT_BTF(x) FLEXCAN_CBT_BTF(x)
+#define CAN_CBT_BTF(x)                             FLEXCAN_CBT_BTF(x)
 /*! @} */
 
 /*! @name IMASK3 - Interrupt Masks 3 Register */
 /*! @{ */
 
-#define CAN_IMASK3_BUF95TO64M_MASK  FLEXCAN_IMASK3_BUF95TO64M_MASK
-#define CAN_IMASK3_BUF95TO64M_SHIFT FLEXCAN_IMASK3_BUF95TO64M_SHIFT
+#define CAN_IMASK3_BUF95TO64M_MASK                 FLEXCAN_IMASK3_BUF95TO64M_MASK
+#define CAN_IMASK3_BUF95TO64M_SHIFT                FLEXCAN_IMASK3_BUF95TO64M_SHIFT
 
-#define CAN_IMASK3_BUF95TO64M(x) FLEXCAN_IMASK3_BUF95TO64M(x)
+#define CAN_IMASK3_BUF95TO64M(x)                   FLEXCAN_IMASK3_BUF95TO64M(x)
 /*! @} */
 
 /*! @name IFLAG3 - Interrupt Flags 3 Register */
 /*! @{ */
 
-#define CAN_IFLAG3_BUF95TO64_MASK  FLEXCAN_IFLAG3_BUF95TO64_MASK
-#define CAN_IFLAG3_BUF95TO64_SHIFT FLEXCAN_IFLAG3_BUF95TO64_SHIFT
+#define CAN_IFLAG3_BUF95TO64_MASK                  FLEXCAN_IFLAG3_BUF95TO64_MASK
+#define CAN_IFLAG3_BUF95TO64_SHIFT                 FLEXCAN_IFLAG3_BUF95TO64_SHIFT
 
 /*! @name IMASK4 - Interrupt Masks 4 Register */
 /*! @{ */
 
-#define CAN_IMASK4_BUF127TO96M_MASK  FLEXCAN_IMASK4_BUF127TO96M_MASK
-#define CAN_IMASK4_BUF127TO96M_SHIFT FLEXCAN_IMASK4_BUF127TO96M_SHIFT
+#define CAN_IMASK4_BUF127TO96M_MASK                FLEXCAN_IMASK4_BUF127TO96M_MASK
+#define CAN_IMASK4_BUF127TO96M_SHIFT               FLEXCAN_IMASK4_BUF127TO96M_SHIFT
 
-#define CAN_IMASK4_BUF127TO96M(x) FLEXCAN_IMASK4_BUF127TO96M(x)
+#define CAN_IMASK4_BUF127TO96M(x)                  FLEXCAN_IMASK4_BUF127TO96M(x)
 /*! @} */
 
 /*! @name IFLAG4 - Interrupt Flags 4 Register */
 /*! @{ */
 
-#define CAN_IFLAG4_BUF127TO96_MASK  FLEXCAN_IFLAG4_BUF127TO96_MASK
-#define CAN_IFLAG4_BUF127TO96_SHIFT FLEXCAN_IFLAG4_BUF127TO96_SHIFT
+#define CAN_IFLAG4_BUF127TO96_MASK                 FLEXCAN_IFLAG4_BUF127TO96_MASK
+#define CAN_IFLAG4_BUF127TO96_SHIFT                FLEXCAN_IFLAG4_BUF127TO96_SHIFT
 
-#define CAN_IFLAG4_BUF127TO96M(x) FLEXCAN_IFLAG4_BUF127TO96M(x)
+#define CAN_IFLAG4_BUF127TO96M(x)                  FLEXCAN_IFLAG4_BUF127TO96M(x)
 /*! @} */
 
 /* The count of CAN_CS */
-#define CAN_CS_COUNT_MB8B (96U)
+#define CAN_CS_COUNT_MB8B                          (96U)
 
 /* The count of CAN_ID */
-#define CAN_ID_COUNT_MB8B (96U)
+#define CAN_ID_COUNT_MB8B                          (96U)
 
 /* The count of CAN_WORD */
-#define CAN_WORD_COUNT_MB8B (96U)
+#define CAN_WORD_COUNT_MB8B                        (96U)
 
 /* The count of CAN_WORD */
-#define CAN_WORD_COUNT_MB8B2 (2U)
+#define CAN_WORD_COUNT_MB8B2                       (2U)
 
 /* The count of CAN_CS */
-#define CAN_CS_COUNT_MB16B (63U)
+#define CAN_CS_COUNT_MB16B                         (63U)
 
 /* The count of CAN_ID */
-#define CAN_ID_COUNT_MB16B (63U)
+#define CAN_ID_COUNT_MB16B                         (63U)
 
 /* The count of CAN_WORD */
-#define CAN_WORD_COUNT_MB16B (63U)
+#define CAN_WORD_COUNT_MB16B                       (63U)

```

Comment:
pls don't introduce unnecessary reformatting on this header and keep the same format as the rest of the mcux headers for easier diff against each other, when needed

Reply:
ok, I removed this commit